src/ex1/index.html

```diff
@@ -1,14 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Exercise 1</title>
-    <link rel="stylesheet" href="style.css" />
+  <title>TODOOPS</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;700&family=Indie+Flower&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-
-<h1>Todo list</h1>
-
-
+  <main class="app-window">
+    <h1 class="app-title">TODOOPS</h1>
+    <article class="todo-content">
+      <form id="add-todo">
+        <input type="text" id="new-todo-title" autofocus required minlength="4" placeholder="Add your new todo" />
+        <button type="submit" id="add-todo-button">+</button>
+      </form>
+      <img id="zero-todos-image" src="../assets/pointer.png" width="100px">
```

Comment:
I find it simpler to control the size of an image by wrapping it with a div, assigning 100% width & height to the image, and controlling the actual size from the div

Reply:
done